examples/custom_totp_generation/main.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+/*
+ In this example, we want to show the definition of custom-generated totp
+*/
+func main() {
+
```

Comment:
looks like no example here... so remove the file for now :) ?

Reply:
:) yes please, I'll add it soon.